lib/addons/src/make-decorator.ts

```diff
@@ -31,13 +31,13 @@ interface MakeDecoratorOptions {
   wrapper: StoryWrapper;
 }
 
-export const makeDecorator: MakeDecoratorResult = ({
+export const makeDecorator = ({
   name,
   parameterName,
   wrapper,
   skipIfNoParametersOrOptions = false,
   allowDeprecatedUsage = false,
```

Comment:
Both `skipIfNoParametersOrOptions` and `allowDeprecatedUsage` have default values set here, so you shouldn't have to set those in the tests explicitly. But I leave it up to you.

Reply:
Ok, I understand why I had to explicitly set them in test, just because they are not declared as optional in `MakeDecoratorOptions`. I will make them optional and revert my changes on tests.